starter/src/main/java/com/vivy/springdoc/SwaggerBlockHoundIntegration.java

```diff
@@ -0,0 +1,22 @@
+package com.vivy.springdoc;
+
+import com.google.auto.service.AutoService;
+import reactor.blockhound.BlockHound;
+import reactor.blockhound.integration.BlockHoundIntegration;
+
+
+@AutoService(BlockHoundIntegration.class)
+public class SwaggerBlockHoundIntegration implements BlockHoundIntegration {
```

Comment:
do you require a test?

Reply:
not really, this is instruction for other dependents